src/GiftedChat.tsx

```diff
@@ -806,7 +806,7 @@ class GiftedChat<TMessage extends IMessage = IMessage> extends React.Component<
 
   renderChatFooter() {
     if (this.props.renderChatFooter) {
-      return this.props.renderChatFooter()
+      return this.props.renderChatFooter(this.props)
```

Comment:
Hi, thanks, can you please add TS definition of this props?

Reply:
I am new to TypeScript. However, I feel we may not need the TS definition for this props. If required, where do we need to write it.